db/migrate/20180320065800_add_address_to_diaper_drive_participants.rb

```diff
@@ -0,0 +1,5 @@
+class AddAddressToDiaperDriveParticipants < ActiveRecord::Migration[5.1]
+  def change
+    add_column :diaper_drive_participants, :address, :string
+  end
+end
```

Comment:
Since these are both part of the same feature, can you combine both those migrations into one? You may need to rollback both, delete one, add to the other, then re-run.